webapp/src/ts/services/lineage-model-generator.service.ts

```diff
@@ -33,12 +36,16 @@ export class LineageModelGeneratorService {
       });
   }
 
+  contact(id, options?) {
+    return this.ngZone.runOutsideAngular(() => this._contact(id, options));
+  }
+
   /**
    * Fetch a contact and its lineage by the given uuid. Returns a
    * contact model, or if merge is true the doc with the
    * lineage inline.
    */
-  contact(id, { merge=false }={}) {
+  private _contact(id, { merge=false }={}) {
```

Comment:
I think by reaching this comment we probably have an agreement :) I'm just going to mark the places to not forget.

In this file apply to:
- _contact
- _report
- _reportSubjects